src/Auth0.php

```diff
@@ -93,11 +94,16 @@ public function login(
         ?string $redirectUrl = null,
         ?array $params = null
     ): string {
+        if (! $this->configuration()->usingStatefulness()) {
```

Comment:
I've added a `usingStatefulness()` helper method on the `Auth0\SDK\Configuration\SdkConfiguration` class, which simply returns `true` if the host application has configured a stateless application strategy.

These throw a ConfigurationException if a method dependant on sessions/statefulness is invoked. (This is not a breaking change as this would have naturally thrown an exception from being inaccessible — just a cleaner exception type is thrown now.)